apps/innbyggerpanelet/src/styles.scss

```diff
@@ -1 +1 @@
-/* You can add global styles to this file, and also import other style files */
+@import './app/app.module.scss';
```

Comment:
Not entirely sure if this does anything. Besides, css-modules should ideally only be imported in individual components and this file rather be used for globals and css variable declarations.

Reply:
Think this is an relic from how i started to write the css, will remove